markdown/1.x/en/documentation/communication.md

```diff
@@ -50,10 +50,10 @@ now to be made to the shop system now. Change the URL settings in `ff-communicat
 ```
 
 
-#### 2. Forward the HTTP Request to FACT-Finder
+#### 2. Forward the HTTP Request to FactFinder
 
 ---
-The request made by FACT-Finder Web Components will look something like: 
+The request made by FactFinder Web Components will look something like: 
 
     `http://domain.com/FACT-Finder/Search.ff?query=shoe&channel=shoeshop_en`.
```

Comment:
I intentionally skipped replacing `FACT-Finder` in the URLs. Please confirm if that's expected.

Reply:
Yes, I think this was the right thing to do.